.github/workflows/backend.yml

```diff
@@ -73,41 +70,65 @@ jobs:
         path: |
           client/python/venv/*
         key: python-client-${{ runner.os }}-pip-${{ hashFiles('client/python/requirements.txt') }}-${{ hashFiles('client/python/test/requirements.txt') }}
+    - uses: actions/cache@v3
```

Comment:
How do the backend tests depend on the front-end being built?

Reply:
If you install gradio from source and you try to run a simple gradio demo, you will get errors if the frontend is not built